package.json

```diff
@@ -1,27 +1,17 @@
 {
   "name": "ui-monorepo",
   "version": "1.0.0",
-  "private": true,
   "scripts": {
     "format:check": "prettier --check .",
-    "lint": "eslint --cache .",
-    "postinstall": "patch-package"
+    "postinstall": "patch-package",
+    "lint": "eslint --cache ."
```

Comment:
Is this order correct?

<img width="320" alt="image" src="https://github.com/user-attachments/assets/9dd457e0-d219-4d19-8b6c-101c4cd5814d">


it seems to me `lint` should go before `postinstall`

Reply:
The logic used to sort aims to place all the pre- and post- scripts around the target script. Therefore, `postinstall` would go right after `install`, which goes before `lint`. If all the scripts were present, the order should be:

- preinstall
- install
- postinstall
- prelint
- lint
- postlint

Some are scripts are not defined in this case but the order reflects that logic.

Reply:
I know the logic of pre/post around their scripts, it seems to me that if the "main" script (`install` in this case) is missing, the pre/post should go as a regular script, sorted alphabetically. But I guess for that I should make my own package 😆